assembly/src/library/path.rs

```diff
@@ -32,6 +32,8 @@ impl LibraryPath {
     /// Path for an executable module.
     pub const EXEC_PATH: &str = "#exec";
 
+    pub const MOD: &str = "mod";
```

Comment:
If we remove the check mentioned in the previous comment, this would probably need to go into `MaslLibrary`.